tools/releaseTools.psm1

```diff
@@ -341,21 +341,31 @@ function Get-ChangeLog
     PrintChangeLog -clSection $clBreakingChange -sectionTitle 'Breaking Changes'
```

Comment:
@adityapatwardhan, your last commit had 2 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Expected exactly '127.0.0.1', but got 10.79.4.115.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 83
83:             $result[0].Address | Should -BeExactly $realAddress
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected strings to be the same, but they were different.
String lengths are both 14.
Strings differ at index 6.
Expected: 'Mac-1439.local'
But was:  'Mac-1449.local'
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 256
256:             $result[0].Source | Should -BeExactly $hostName
```